Source/Core/DolphinQt/Settings/BroadbandAdapterSettingsDialog.h

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <QDialog>
+
+class QDialog;
```

Comment:
Since you're including <QDialog> you don't need to forward declare it.

Reply:
Done